<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>641</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Sequence</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>20</y>
        <width>613</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="EnterCyclesHerelabel_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>Enter Cycles Here</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="Executinglabel_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Executing</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="polonatorStart">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QTextEdit" name="polonatorCycleList">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="cycleTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="rowCount">
            <number>10</number>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>71</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>71</number>
           </attribute>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <column>
            <property name="text">
             <string>Primer</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Direction</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Position</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="polonatorCycleEntryValidate_2">
         <property name="text">
          <string>Validate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>611</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="sequenceGraphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>611</width>
        <height>101</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="clearSelectionPB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Selection</string>
      </property>
     </widget>
     <widget class="QComboBox" name="touchFlagCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Operate Polonator via Graphical Interface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Operate Polonator via Touch Sensor</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Maintenence</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>301</width>
        <height>66</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="stopButton_2">
         <property name="text">
          <string>Abort All Processes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="homing_axis_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Include Homing Axis</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_34">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>501</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_99">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Lock</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_100">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Unlock</string>
       </property>
      </widget>
      <widget class="QSlider" name="stageSwitch_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>20</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_107">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Stage</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>383</width>
         <height>222</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="SelectFlowcelllabel_6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Select Flowcell</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_21">
              <item>
               <widget class="QRadioButton" name="stagealign_fc0_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Flowcell 0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="stagealign_fc1_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Flowcell 1</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="Selectlanelabel_3">
              <property name="text">
               <string>Select Lane</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="stagealign_lane_3">
              <property name="maximum">
               <number>7</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <layout class="QGridLayout" name="gridLayout_40">
            <item row="0" column="0">
             <widget class="QPushButton" name="utilsHomeButton_5">
              <property name="text">
               <string>Home / reset motion axes</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="stagealign_gotoposition_3">
              <property name="text">
               <string>Goto alignment position</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="utilsStatusButton_5">
              <property name="text">
               <string>Get motion status</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="stagealign_viewlog_3">
              <property name="text">
               <string>View stage alignment log</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="stagealign_dostagealign_3">
            <property name="text">
             <string>Perform stage alignment now</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>501</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_97">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter Wheel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_105">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>89</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Unlock</string>
       </property>
      </widget>
      <widget class="QSlider" name="filterSwitch_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>49</y>
         <width>20</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_104">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Lock</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_20">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_38">
          <item>
           <widget class="QLabel" name="label_89">
            <property name="text">
             <string>Position:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_90">
            <property name="text">
             <string>Filter Type:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_39">
          <item row="0" column="0">
           <widget class="QLabel" name="label_91">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_92">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_93">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_94">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_95">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_96">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGraphicsView" name="graphicsView_13"/>
          </item>
          <item row="1" column="1">
           <widget class="QGraphicsView" name="graphicsView_14"/>
          </item>
          <item row="1" column="2">
           <widget class="QGraphicsView" name="graphicsView_15"/>
          </item>
          <item row="1" column="3">
           <widget class="QGraphicsView" name="graphicsView_16"/>
          </item>
          <item row="1" column="4">
           <widget class="QGraphicsView" name="graphicsView_17"/>
          </item>
          <item row="1" column="5">
           <widget class="QGraphicsView" name="graphicsView_18"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="stagealign_viewcurrent_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>570</y>
        <width>143</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>View current image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stagealign_viewscore_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>620</y>
        <width>135</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>View score matrix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stagealign_viewbase_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>660</y>
        <width>126</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>View base image</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPro_Mode"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPro_Mode">
   <property name="text">
    <string>Pro Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
